testsuite/tests/singlecluster/authorino/identity/anonymous/test_anonymous_identity.py

```diff
@@ -31,8 +37,9 @@ def test_anonymous_identity(client, auth, authorization):
     response = client.get("/get")
     assert response.status_code == 401
 
+    generation = authorization.generation
     authorization.identity.add_anonymous("anonymous")
-    authorization.wait_for_ready()
+    authorization.wait_until(has_observed_generation(generation + 1))
```

Comment:
You still need the `wait_for_ready` as just waiting for Authorino to pick up the the AuthPolicy is not enough. The observedGeneration can be incremented but the Policy can be in status "Not Enforced". `wait_for_ready` makes sure the Policy is also "Enforced"